File: Python/Activities/Project/Activity2.py
from selenium import webdriver
from selenium.webdriver.common.by import By

driver = webdriver.Chrome()


def navigateToURL():
    options = webdriver.ChromeOptions()
    options.add_experimental_option("detach", True)
    # driver = webdriver.Chrome(options=options)
    driver.get('https://alchemy.hguy.co/lms')
    get_title = driver.title
    print('Page title is: ', get_title)
    assert get_title == 'Alchemy LMS – An LMS Application'
    print('Assertion successful, test completed!')


def verifyTheHeadingText():
    headingText = driver.find_element(By.CLASS_NAME, 'uagb-ifb-title').text;
    assert headingText == 'Learn from Industry Experts'
    print('Heading Text is: ', headingText)


def closeBrowser():
    driver.quit()


navigateToURL()
verifyTheHeadingText()
closeBrowser()

File: Selenium/Project/Activity2.java
package LMSActivities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;
import org.testng.Assert;
import org.testng.annotations.AfterMethod;
import org.testng.annotations.BeforeMethod;
import org.testng.annotations.Test;

public class Activity2 {
    WebDriver driver;

    @BeforeMethod
    public void beforeMethod() {
        //Create a new instance of the Firefox driver
        driver = new FirefoxDriver();

        //Open browser
        driver.get("https://alchemy.hguy.co/jobs/");
    }

    @Test
    public void verifyTheHeadingText() {
        // Check the heading of the page
        String headingText=driver.findElement(By.className("entry-title")).getText();

        //Print the heading of the page
        System.out.println("Page Heading is: " + headingText);

        //Assertion for heading page
        Assert.assertEquals("Welcome to Alchemy Jobs", headingText);
    }

    @AfterMethod
    public void afterMethod() {
        //Close the browser
        driver.quit();
    }
}
